mailpoet/lib/Automation/Engine/Data/Subject.php

```diff
@@ -27,10 +27,15 @@ public function getArgs(): array {
     return $this->args;
   }
 
+  public function hash(): string {
+    return md5($this->getKey() . Json::encode($this->getArgs()));
```

Comment:
I think for the use case of hashing, it's safer to use `serialize` over JSON. The reason is that JSON can (in theory) contain arbitrary whitespace, have arbitrary order on object keys, etc.

Reply:
ed7f38ade